pyOCD/flash/flash_k82f25615.py

```diff
@@ -0,0 +1,79 @@
+"""
```

Comment:
I didn't see an algo for this guy in the FlashAlgo repo. Could you add one for this?